src/main/kotlin/blackjack/domain/Hand.kt

```diff
@@ -0,0 +1,43 @@
+package blackjack.domain
+
+class Hand(
+    cards: List<Card> = emptyList(),
+) {
+    val cards = mutableListOf<Card>()
+
+    init {
+        this.cards.addAll(cards)
+    }
```

Comment:
아래와 같이 구현한다면 굳이 init 블럭을 활용하지 않아도 되겠네요!

```suggestion
class Hand(
    cards: List<Card> = emptyList(),
) {
    val cards: MutableList<Card> = cards.toMutableList()
```

Reply:
감사합니다 🙇